advanced/Exercise_3.md

```diff
@@ -64,7 +64,7 @@ git commit -m "Add README file"
 ```
 ## Generate the README file in HTML <a name="html"></a>
 
-It may be useful to convert the *README* file to HTML in order to see how it will look to a user browsing your repository on the web. We will use a document converter called [Pandoc](https://pandoc.org) to do this. You can check if you have Pandoc installed by running the following command:
+It may be useful to convert the *README* file to HTML in order to see how it will look to a user browsing your repository on the web. We will use a document converter called [Pandoc](https://pandoc.org) to do this. If you use conda, then `conda install -c conda-forge pandoc` should do the trick. You can check if you have Pandoc installed by running the following command:
```

Comment:
Doesn't this belong to the install instructions below? Here they should only check if they have it installed.

Reply:
I think we should mention first how to install it before actually testing `pandoc -h`. I made a rearrangement here [29069e3](https://github.com/C2SM/git-course/commit/29069e36947f1f1813a2bf18c6c7d945e9daaf08)

Reply:
That also works